### PR TITLE
Enable `no-nested-ternary` lint rule

### PR DESCRIPTION
**Pull request by iamacook**
## Summary

Nested ternaries are hard to follow. As such, this enabled the [`no-nested-ternary` lint rule](https://eslint.org/docs/latest/rules/no-nested-ternary) and solves the only flouting occurance.

## Changes

- Enable `no-nested-ternary` lint tule.
- Change nested ternary to use `if` clauses.

### CODE REVIEW

File: src/routes/transactions/mappers/common/twap-order.mapper.ts [hunk 1]
```diff
@@ -91,14 +91,18 @@ export class TwapOrderMapper {
...
-    // Fetch all order parts if the transaction has been executed, otherwise none
-    const partsToFetch = transaction.executionDate
-      ? hasAbundantParts
-        ? // We use the last part (and only one) to get the amounts/fees of the entire
-          // order and we only need one to get the token info
-          twapParts.slice(-1)
-        : twapParts
-      : [];
+    // Fetch no parts if transaction isn't executed, otherwise only those up to max
+    const partsToFetch = ((): Array<GPv2OrderParameters> => {
```
Comment (PooyaRaki) [inline src/routes/transactions/mappers/common/twap-order.mapper.ts:95]: Nit: Can we write it without using a function?
Calling a function adds another layer to the execution context.

Comment (iamacook) [reply src/routes/transactions/mappers/common/twap-order.mapper.ts]: Changed in 12ff8489ce6331f9765d6f1d3751c9314d4b0483.

**Review by hectorgomezv — APPROVED** [review]
